### PR TITLE
Start working on part of website

### PR DESCRIPTION
**Pull request by TH1Tprod**
Creating new version of website. Redesign, optimazing code...

### CODE REVIEW

File: web/bs.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+browser-sync start --server --files "*.*"
```
Comment (hbvhuwe) [inline web/bs.sh]: What is this for?

File: web/index.html [hunk 2]
```diff
@@ -12,17 +12,62 @@
...
+
 	<div class="container">
 		<div class="find-tickets">
 			<div class="field">
-				<div class="air-from"></div>
-				<div class="air-where"></div>
-				<div class="date-from"></div>
-				<div class="date-back"></div>
+				<div class="air-from">
+					<input type="text" class="inp-from" placeholder="From...">
```
Comment (hbvhuwe) [inline web/index.html]: There you use English, but in other parts of document you using Russian. Please use one language or setup translation.

File: web/js/funcbtn.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// Get the modal
```
Comment (hbvhuwe) [inline web/js/funcbtn.js]: Very strange name for file. It is impossible to understand what is in this file by its name. Name it something like "dialogHandler.js"

File: web/bs.sh (hunk 1 above)
Comment (TH1Tprod) [reply web/bs.sh]: Secret!

Comment (hbvhuwe) [reply web/bs.sh]: So, reveal this secret to me! :smiling_imp:

**Pull request merged**